recipes/nss_wrapper/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.1.11" %}
+
+package:
+  name: nss_wrapper
+  version: {{ version }}
+
+source:
+  url: https://ftp.samba.org/pub/cwrap/nss_wrapper-{{ version }}.tar.gz
+  sha256: fb45332e7fb4d85d3ea829ed18172366196c17cbd32ef690a7e1309f0c37b4e0
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - make
```

Comment:
don't you need ``nss`` to run and compile this?

Reply:
No. It's designed to be used as `LD_PRELOAD=libnss_wrapper.so my_executable` to make it possible to change or edit the `passwd` and `group` files without affecting the host system (or requiring root privileges).

See https://cwrap.org/nss_wrapper.html